components/camel-casper/src/main/docs/doc.md

```diff
@@ -0,0 +1,259 @@
+# Camel casper producer component
+Camel Casper component for interacting with Casper network using RPC calls (Java SDK)
+
```

Comment:
The documentation file needs to be named like the component and need to be an .adoc, take a look at other components to copy the base skeleton